system-parachains/asset-hubs/asset-hub-kusama/src/staking/nom_pools.rs

```diff
@@ -41,8 +41,7 @@ impl pallet_nomination_pools::Config for Runtime {
 	type MaxUnbonding = <Self as pallet_staking_async::Config>::MaxUnlockingChunks;
 	type PalletId = PoolsPalletId;
 	type MaxPointsToBalance = MaxPointsToBalance;
-	type WeightInfo = (); // TODO @ggwpez cannot do this until the nomination pools pallet is fixed to be benchmarkable
-					   // with pallet-staking-async.
+	type WeightInfo = (); // Have to use stock weights since nom-pools is not benchmarkable with pallet-staking-async.
```

Comment:
```suggestion
    // TODO: https://github.com/paritytech/polkadot-sdk/pull/9016
    // Have to use stock weights since nom-pools is not benchmarkable with pallet-staking-async.
	type WeightInfo = ();
```